### ISSUE TITLE
Slider not showing images in full

### ISSUE THREAD

**Issue by NXTGENDESIGN** [Dogfalo/materialize#559]
![screen shot 2015-01-29 at 4 49 58 pm](https://cloud.githubusercontent.com/assets/8184897/5961888/e9c961b4-a7df-11e4-9e86-d4f2c4047885.png)

**Comment by acburst** [comment]
Are you using the latest code? there was a fix that resized slider images in one of the recent releases

**Comment by liorzoue** [comment]
Same here with 0.95.1 - Chrome OS X latest

Reloading (F5) the page : 
![capture d ecran 2015-01-31 a 01 39 48](https://cloud.githubusercontent.com/assets/3919250/5985909/1fd07b52-a8ea-11e4-90e3-03f522e7b49a.png)

From a link :
![capture d ecran 2015-01-31 a 01 39 56](https://cloud.githubusercontent.com/assets/3919250/5985908/1acba820-a8ea-11e4-9fee-3f2bf5f7837f.png)

My markup :
```html
<body>
 <div class="slider fullscreen">
  <ul class="slides">
   <li>
    <img src="../img/galerie/image1.jpg">
   </li>
   <li>
    <img src="../img/galerie/image2.jpg">
   </li>
   <li>
    <img src="../img/galerie/image3.jpg">
   </li>
  </ul>
 </div>
</body>
```

**Comment by Dontorpedo** [comment]
same here with 0.95.1 firefox - latest , it resizes after a pageload (sometimes), but not allways

**Comment by acburst** [comment]
Should be fixed in 8eebca06cb6a353e77c4377f76258aaf4ea94fb1
